packages/react-scripts/utils/createJestConfig.js

```diff
@@ -28,6 +28,10 @@ module.exports = (resolve, rootDir, isEjecting) => {
     setupTestFrameworkScriptFile: setupTestsFile,
     testPathIgnorePatterns: ['<rootDir>/(build|docs|node_modules)/'],
     testEnvironment: 'node',
+    preprocessorIgnorePatterns: ["<rootDir>/node_modules"],
+    haste: {
+        providesModuleNodeModules: [".*"],
+    },
   };
```

Comment:
It's sort of unfortunate that this will end up in config after ejecting.
(@cpojer do we have better options?)